settings/DevHome.Settings/Views/AccountsPage.xaml.cs

```diff
@@ -58,7 +60,24 @@ private async void AddAccountButton_Click(object sender, RoutedEventArgs e)
         var numProviders = ViewModel.AccountsProviders.Count;
         if (numProviders == 1)
         {
-            await ShowLoginUIAsync("Settings", this, ViewModel.AccountsProviders[0]);
+            if (ViewModel.AccountsProviders[0].DeveloperIdProvider.GetAuthenticationExperienceKind() == Microsoft.Windows.DevHome.SDK.AuthenticationExperienceKind.CardSession)
+            {
+                await ShowLoginUIAsync("Settings", this, ViewModel.AccountsProviders[0]);
+            }
+            else if (ViewModel.AccountsProviders[0].DeveloperIdProvider.GetAuthenticationExperienceKind() == Microsoft.Windows.DevHome.SDK.AuthenticationExperienceKind.CustomProvider)
+            {
+                IntPtr windowHandle = IntPtr.Zero;
+                foreach (Process activeProcess in Process.GetProcesses())
+                {
+                    if (activeProcess.MainWindowTitle.Contains("Dev Home"))

```

Comment:
`MainWindowTitle` can [throw an exception](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.process.mainwindowtitle?view=net-7.0#exceptions)

Reply:
Addressed in recent commit. Thanks!